pandas/core/internals.py

```diff
@@ -1845,12 +1845,13 @@ def is_bool(self):
         """
         return lib.is_bool_array(self.values.ravel())
 
-    # TODO: Refactor when convert_objects is removed since there will be 1 path
```

Comment:
Remove `fn_kwargs`?  `convert` appears to be just a wrapper around all of the object conversion functions as I mentioned above.  Doesn't that allow us to pass in keyword arguments specific to each of those functions?